smqtk_classifier/impls/classifier/index_label.py

```diff
@@ -3,10 +3,10 @@
 import numpy as np
 from smqtk_dataprovider import from_uri
 
-from smqtk_classifier.interfaces.classifier import Classifier
+from smqtk_classifier.interfaces.classify_descriptor import ClassifyDescriptor
 
 
-class IndexLabelClassifier (Classifier):
+class IndexLabelClassifier (ClassifyDescriptor):
```

Comment:
so because all current implementations are used for descriptors, place them all into a descriptor dir?

Reply:
Roughly. The pattern so far is that the `impls` subdirectory nearly matches the name of the interface, e.g. the previous interface was named `Classifier`, so there was a sub-directory `classifier` into which implementations of that interface were placed. Now, we have renamed that interface to `ClassifyDescriptor` so it would follow that we would also change this directory to maybe be `classify_descriptor`, etc. We don't have any impls for the new `ClassifyImage` interface, but they would need to go somewhere, and I would imagine we would not ideally want to mix them with the descriptor-based impls, maybe in a separate subdirectory `classify_image`.